spec/index.bs

```diff
@@ -1954,6 +1965,59 @@ interface GPUAdapterInfo {
     </div>
 </div>
 
+### Behaviors ### {#behaviors}
+
+A <dfn dfn>behavior</dfn> is a set of optional WebGPU functionality that may not be supported
+on all implementations, typically due user agent limitations.
+
+Functionality that is part of a behavior is available if the user agent supports it.
+In particular, there is no mechanism for an application to request that a behavior be enabled.
+
+<div algorithm>
```

Comment:
Not sure this part is needed.

Reply:
I don't think it is, especially since the dfn isn't used. My editorial style is that rigor is not needed when there's already no room for misinterpretation.